packages/core/ts/Poll.ts

```diff
@@ -139,6 +141,74 @@ export class Poll implements IPoll {
   // how many users signed up
   private numSignups = 0n;
 
+  /**
+   * Save the current state of the poll to a file
+   * @param filePath - The path to save the state to
+   */
+  saveState = async (filePath: string): Promise<void> => {
```

Comment:
No need to implement new functions (save and load). There are `toJSON` and `fromJSON` that can be updated and used for this.